fixtures/glp/impression-x4-bar-10.json

```diff
@@ -0,0 +1,994 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Impression X4 Bar 10",
+  "categories": ["Moving Head", "Color Changer", "Effect", "Strobe"],
```

Comment:
Remove 'Moving Head' -> make register again!

Reply:
But isn't it a moving head?